src/fiatconnect/consts.ts

```diff
@@ -34,7 +34,7 @@ export const FIATCONNECT_CURRENCY_TO_WALLET_CURRENCY: Record<FiatType, LocalCurr
   [FiatType.ZAR]: LocalCurrencyCode.ZAR,
 }
 
-export const WALLET_CRYPTO_TO_FIATCONNECT_CRYPTO: Record<CiCoCurrency, CryptoType> = {
+export const WALLET_CRYPTO_TO_FIATCONNECT_CRYPTO: Partial<Record<CiCoCurrency, CryptoType>> = {
```

Comment:
```suggestion
export const WALLET_CRYPTO_TO_FIATCONNECT_CRYPTO: Record<CiCoCurrency, CryptoType | null> = {
```

this way, we are still forced to consider whether a mapping should be added, but we can add "null" if no sensible mapping exists